App.js

```diff
@@ -17,14 +18,29 @@ import AppNavigator from './navigation/AppNavigator';
 
 Amplitude.initialize(keys.AMPLITUDE);
 
-if (__DEV__) {
-  // override amplitude tracking
+const channel = Constants?.manifest?.releaseChannel;
+const emptyStorage = () =>
+  AsyncStorage.multiRemove(['@posts', '@media', '@groups', '@missions']);
+// override amplitude tracking
+if (channel === null || channel === undefined || channel === '') {
   Amplitude.logEvent = name => console.log(`[amplitude]: ${name}`);
   Amplitude.logEventWithProperties = (name, data) =>
     console.log(`[amplitude]: ${name}\n`, data);
-
-  // Erase all AsyncStorage
-  AsyncStorage.multiRemove(['@posts', '@media', '@groups', '@missions']);
+  emptyStorage();
+} else if (channel.indexOf('develop') !== -1) {
+  // beta testing from the store--we want to log this to amplitude, but
+  //  separate it out
+  Amplitude.logEvent = name => {
+    name = `BETA ${name}`;
+    console.log(`[amplitude]: ${name}`);
+    Amplitude.logEvent(name);
```

Comment:
nice! so we'll just log the event with a `BETA` tag 🔥 
that works for me 👍 
should we just remove the `console.log`s tho?

Reply:
@dereckquock I kind of want it to log in the console unless we're release.